app/models/code_school.rb

```diff
@@ -2,4 +2,22 @@ class CodeSchool < ApplicationRecord
   validates :name, :url, :logo, presence: true
   validates_inclusion_of :full_time, :hardware_included, :has_online, :online_only, :in => [true, false]
   has_many :locations, -> { order('state ASC, city ASC') }, dependent: :destroy
+
+  before_create :set_valid_scheme, :if => Proc.new{ |cs| URI(cs.url).scheme.nil? }
+
+  private
+
```

Comment:
Would you please correct the linter errors. 
https://travis-ci.org/OperationCode/operationcode_backend/builds/452053452#L838